src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DhGaugeParameters.hpp

```diff
@@ -0,0 +1,132 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <string>
+
+#include "Options/Options.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+/*!
```

Comment:
Can you format this as follows (missing brief, removed extra spaces)
```cpp
/*!
 * \brief A struct holding the parameters for initializing damped harmonic gauge
 *
 * If UseRollon is true, the gauge transitions ("rolls on") to damped harmonic
 * from determines whether the gauge transitions to damped harmonic ("rolls on")
 * from the initial data gauge; otherwise, the gauge begins immediately in
 * damped harmonic gauge.
 */
```